paper/paper.md

```diff
@@ -70,40 +70,43 @@ for species, agents in grouped:
 ```
 
 ### Spaces
-Mesa 3 provides multiple types of spaces for agent interactions, broadly categorized into discrete (cell-based) and continuous spaces. Each type serves different modeling needs while maintaining a consistent API.
+Mesa 3 provides multiple spatial environments where agents interact, offering both discrete (cell-based) and continuous space implementations:
 
 #### Discrete Spaces
```

Comment:
you are indexing into level 4, I would just remove this heading.

Reply:
Seeing the rendered paper I agree, done.